.gitignore

```diff
@@ -4,4 +4,4 @@ vendor/*
 *.cpp.bin*
 update.sh
 push.sh
-.vscode/
+.vscode/
```

Comment:
Did you intend to remove the newline at the end of this file? Probably not, but it should have shown up in your commits as a change.

Reply:
I was trying to get it to match on my end so it wouldn't change, but it seems I have chosen... poorly. I think I remember the newline being shown as a change on my end, so I got confused.

Reply:
Can you add a newline and then commit/push again?

Reply:
Looks like you took care of it.